README.md

```diff
@@ -107,6 +107,13 @@ line.
   CoreOS image is being installed on.  Optional; defaults to `metal`.
   Normally this should be specified only if installing inside a virtual
   machine.
+* `coreos.inst.save_label` - Comma-separated labels of partitions to
```

Comment:
If it's too late to change this, that's OK, but I think these would be less ambiguous with `part` somewhere in their names (and the corresponding CLI switches). E.g. `coreos.inst.save_partlabel` and `coreos.inst.save_partindex`.

Reply:
Not too late, but the problem with the command-line arguments is the usual one: 80-character limit.  And the kargs should probably match the command line.  Any ideas for shorter names?

Reply:
Okay, updated to `{--,}save[-_]part{label,index}` with shorter parameter variables:

```
        --save-partlabel <lx>...    Save partitions with this label glob
        --save-partindex <id>...    Save partitions with this number or range
```